xrpl/models/transactions/amm_bid.py

```diff
@@ -41,14 +41,14 @@ class AMMBid(Transaction):
     The definition for the other asset in the AMM's pool. This field is required.
     """
 
-    bid_min: Optional[Amount] = None
+    bid_min: Optional[IssuedCurrencyAmount] = None
     """
     Pay at least this amount for the slot.
     Setting this value higher makes it harder for others to outbid you.
     If omitted, pay the minimum necessary to win the bid.
     """
 
-    bid_max: Optional[Amount] = None
+    bid_max: Optional[IssuedCurrencyAmount] = None
```

Comment:
Are XRP amounts not valid for these?

This fix should be added to the changelog.

Reply:
Right, only LPToken amounts are accepted. I just merged in a fix for this - #649